Dockerfile

```diff
@@ -0,0 +1,87 @@
+ARG VITE_KEYCLOAK_BASE_URL
+ARG VITE_KEYCLOAK_REALM
+ARG VITE_KEYCLOAK_CLIENT_ID
+ARG VITE_BACKEND_BASE_URL
+
+#################################################
+##                   BASE                       #
+#################################################
+FROM node:20 AS base
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+RUN corepack enable
+
+WORKDIR /app
+RUN npm i -g pnpm --force
+COPY package.json pnpm-lock.yaml pnpm-workspace.yaml ./
+RUN pnpm install
+
+
+#################################################
+##                   BUILD                      #
+#################################################
+FROM base AS build
+
+ARG VITE_KEYCLOAK_BASE_URL
+ARG VITE_KEYCLOAK_REALM
+ARG VITE_KEYCLOAK_CLIENT_ID
+ARG VITE_BACKEND_BASE_URL
+
+COPY tsconfig.base.json tsconfig.json .eslintrc.json ./
+COPY ./libs ./libs
+COPY ./apps/public/ ./apps/public/
+COPY ./apps/admin/ ./apps/admin/
+RUN pnpm install
+RUN pnpm build
+
+
+################################################
+#                   ADMIN                      #
+################################################
+FROM nginx:stable-alpine AS admin_signalen
```

Comment:
We've called the application 'meldingen', right? Let's use that name here as well, and for all other mentions of 'signalen' in this PR.